apps/docs/content/nginx/how-to/customize-web-server.mdx

```diff
@@ -84,3 +84,9 @@ zerops:
 5. Ensure that the <a href="/nginx/how-to/build-pipeline#deployfiles">`build.deployFiles`</a> contains the folder with the `siteConfigPath` or add the path to the Nginx config file to the `deployFiles` list. Zerops will deploy the file to the runtime container(s).
 
 6. [Trigger](/nginx/how-to/trigger-pipeline) the build & deploy pipeline.
+
+### Built-in Prerender.io Support
+
+The default Nginx configuration includes automatic prerender.io support for SEO optimization. When `PRERENDER_TOKEN` is set, Nginx will automatically serve pre-rendered content to search engines and social media crawlers.
+
+See [environment variables](/nginx/how-to/env-variables#prenderio-support) for configuration details.
```

Comment:
the link has a typo `#prenderio-support` -> `#prerenderio-support`

Reply:
Done!